doc/tor.md

```diff
@@ -223,3 +223,40 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world the Tor network is monitored or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor.
+
+- For maximum privacy, it is preferable to disable accepting incoming connections.
+
+- Users can download packages with [torsocks](https://gitlab.torproject.org/legacy/trac/-/wikis/doc/torsocks)
+  for better security and privacy.
+
+- Trade-offs involved in using `onlynet=onion`:
+
+  The `onlynet=onion` configuration option can potentially ensure the node attempts to only
+  connect over Tor. It is more private when you combine it with no reachable IPv4/IPv6 address,
+  in particular if you want to broadcast transactions without them being correlatable with your IP.
```

Comment:
This is bad advice. Tor is a centralised network, and using it as your only network puts the user at risk for no benefit. It isn't more private, as it doesn't do anything to stop the "cookies" (UTXO/address relationships) Bitcoin wallets need to work.

Reply:
I think 'more private' here is a comparison between `onlynet=onion` and `onlynet=onion` + `bind=onion`. This was suggested by @sipa in https://github.com/bitcoin/bitcoin/pull/21157#discussion_r574894711

Reply:
I relate to Luke's feedback and am a bit uncomfortable with the lines of advice he points out; maybe they could use more nuance and context. Otherwise, they seem likely to be misinterpreted or followed somewhat blindly.

Reply:
> maybe they could use more nuance and context

Agree